### PR TITLE
microshift automate lvm provisioning

### PR DESCRIPTION
**Pull request by copejon**
- add lvm device to microshift test instance
- fix step name
- fix dupe step
- update integration-test commands with AAO entrypoint
- update metadata

[USHIFT-208](https://issues.redhat.com//browse/USHIFT-208)

### CODE REVIEW

**Comment by copejon** [comment]
/test 
ci/rehearse/openshift/microshift/main/e2e-openshift-conformance-sig-cli

**Comment by dhellmann** [comment]
This looks good.

How is the new disk deleted when the job is done? Does the teardown remove everything with the INSTANCE_PREFIX?

**Comment by copejon** [comment]
@dhellmann After 2x checking, it looks like the auto-delete flag isn't set for disks created via `create disk`, where it is set if created by the `--create-disk` flag with create instance.  Will fix.

File: ci-operator/step-registry/upi/gcp/rhel8/add-disk/upi-gcp-rhel8-add-disk-commands.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
+gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+
+echo "$(date -u --rfc-3339=seconds) - Creating GCP Persistent Disk ${GCP_DISK_NAME}"
+gcloud compute disks create "${GCP_DISK_NAME}" --size="${GCP_DISK_SIZE}"
+
+# The attached disk will be soft-linked on the host as /dev/disk/by-id/google-${GCP_DISK_NAME}
+# The softlink suffix is provided via --device-name
+echo "$(date -u --rfc-3339=seconds) - Attaching GCP Persistent Disk ${GCP_DISK_NAME} to instance ${INSTANCE_PREFIX}"
+gcloud compute instances attach-disk "${INSTANCE_PREFIX}" \
```
Comment (dhellmann) [inline ci-operator/step-registry/upi/gcp/rhel8/add-disk/upi-gcp-rhel8-add-disk-commands.sh:39]: If create succeeds but this command fails will the disk be orphaned and stick around?

Comment (copejon) [reply ci-operator/step-registry/upi/gcp/rhel8/add-disk/upi-gcp-rhel8-add-disk-commands.sh:39]: You're too quick :D I added a catch for this as a non-test-failing post submit job.

Comment (dhellmann) [reply ci-operator/step-registry/upi/gcp/rhel8/add-disk/upi-gcp-rhel8-add-disk-commands.sh:39]: I'm wearing out the page refresh button over here. :-)

This version looks good. Let me know when you're ready to merge or if you want someone else with more GCP/CI experience to take a look.

Comment (copejon) [reply ci-operator/step-registry/upi/gcp/rhel8/add-disk/upi-gcp-rhel8-add-disk-commands.sh:39]: CI looks happy.  I think it's ready to go.

**Pull request merged**